cmd/feed-dns/main.go

```diff
@@ -67,6 +69,10 @@ func init() {
 		"A label=value pair to attach to metrics pushed to prometheus. Specify multiple times for multiple labels.")
 	flag.IntVar(&awsAPIRetries, "aws-api-retries", defaultAwsAPIRetries,
 		"Number of times a request to the AWS API is retried.")
+	flag.StringVar(&internalIngressFQDN, "internal-ip", "",
```

Comment:
These are expected to be CNAMEs not IPs, so the variable name and help will need updating to reflect this